.github/workflows/pullrequest.yml

```
@@ -276,6 +282,16 @@ jobs:
       - name: Building
         run: ./scripts/ci/run-in-parallel-native.sh build
 
+  docker-build:
+    uses: ./.github/workflows/docker-build.yml
```

actionlint (1.7.4)

input "GIT_BRANCH" is required by "./.github/workflows/docker-build.yml" reusable workflow

(workflow-call)